test/bigtest/network/config.js

```diff
@@ -153,7 +153,9 @@ export default function config() {
   this.get('/erm/refdata/TitleInstance/type', () => {
     return [
       { 'id': '188389636d9ece46016d9ed0184b0027', 'value': 'journal', 'label': 'Journal' },
-      { 'id': '188389636d9ece46016d9ed018550028', 'value': 'book', 'label': 'Book' }
+      { 'id': '188389636d9ece46016d9ed018550028', 'value': 'book', 'label': 'Book' },
+      { 'id': '170227012020671001', 'value': 'serial', 'label': 'Serial' },
```

Comment:
The output from that endpoint (against real backends) for me doesn't have Serial or Monograph entries. Am I missing something? Is that forthcoming? It doesn't look like it's being used.

Reply:
Ah, I thought originally I'd need to add them there in order to use them in my objects, but realised afterwards I didn't have to, so these can go if they need to. I'm surprised that the endpoint doesn't return them for you, because they're 100% type options. I'll remove them for the time being since they're not in use in my test anyway.